src/main/flight/imu.c

```diff
@@ -437,6 +441,13 @@ static float imuCalcKpGain(timeUs_t currentTimeUs, bool useAcc, float *gyroAvera
        }
     }
 
+    if ((crashRecoveryModeActive()) && (currentPidProfile->crash_recovery == PID_CRASH_RECOVERY_LEVEL)) {
+      ret = imuRuntimeConfig.dcm_kp * imuRuntimeConfig.recovery_coef;
+    }
+
+    DEBUG_SET(DEBUG_RECOVERY, 0, crashRecoveryModeActive());
+    DEBUG_SET(DEBUG_RECOVERY, 1, lrintf(ret*100.0f));
```

Comment:
Betaflight coding style is to have spaces around operators (except for the inside of braces).